en/publish/publish-android-instant.md

```diff
@@ -2,25 +2,98 @@
 
 ## Requirement 
  - Android Studio 3.0+
- - Android Phone 8.0+
+
+ - Android Phone 6.0+
+
  - NDK r10c +
- 
-## Publish Process
- 1. Start cocos creator. 
- 2. Open Project->Build panel.
- 3. Click **Record** button to open simulator ,it will record game resources info automatically.
- 
+
+   
+
+## Instant Games Publish Process
+
+ 1. Start CocosCreator. 
+
+ 2. Open **Projec** t-> **Build panel** , select **Android Instant** platform , and **android-instant** template.
+
+ 3. If you don't need to refactor or want to experience the publish process , You can hook the **Skip Record & Refactor **box , then click **Build** , **Compile **, connect android phone to computer and then click **Play** , when process complete you can run instant app on the phone.
```

Comment:
You can hook the **Skip Record & Refactor ** box , then click **Build**, **Compile**, connect android phone to computer and then click **Play**